pkg/skaffold/jib/jib_linux.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
do we really need both jib_linux and jib_darwin? @balopat ?

Reply:
no just call it `jib.go` and put `// +build linux darwin` at the top

Reply:
you can call it `jib_unix.go` if you really want

Reply:
Okay, will go with `jib_unix.go` since `jib.go` has common logic shared with `jib_windows.go`